# MotorPosTar.msg

uint16 num
float64[] pos_tar
float64[] time_dur